docs/installation/DownloadAndInstall.md

```diff
@@ -1,56 +1,205 @@
+in the file name, which indicates a "hidden" file on linux and macOS operating
+systems).
+
+The configuration file contains the following values used by ScubaGoggles when
+running the conformance assessments:
+
+| Name        | Description                                       |
+|-------------|---------------------------------------------------|
+| credentials | Location and name of the Google credentials file. |
+| opa_dir     | Location of the OPA executable.                   |
+| output_dir  | Location of the ScubaGoggles output directory.    |
+
+Run the setup utility with this command:
+
+```shell
+scubagoggles setup
+```
+
+You will be prompted to enter the output directory, location of the OPA
+executable, and the location and name of the Google credentials file.  By
+default, the OPA executable will be downloaded into the directory you specify.
+You do not have to create the Google credentials file before running setup.  If
+the credentials file doesn't exist, you will see a warning indicating that it is
+missing.  You will need to have the credentials file before running the
+conformance assessment.  See the [instructions for creating a credentials
+file](../authentication/AuthenticationMethods.md).
+
+These are sample outputs from running the setup utility before the required
+files are available.  Warnings are shown, but the `.scubagoggles` configuration
+file is still created.
+
+### Windows Example
+
+```
+> scubagoggles setup
+Setup: output directory
+Scubagoggles output directory [C:\Users\userID\scubagoggles]?
+Create directory C:\Users\userID\scubagoggles [Yes/no]?
+  creating: C:\Users\userID\scubagoggles
+  C:\Users\userID\scubagoggles
+Setup: OPA executable directory
+(WARNING): OPA executable not found in PATH
+Location of OPA executable [C:\Users\userID\scubagoggles]?
+  downloading: opa_windows_amd64.exe
+  OPA executable: C:\Users\userID\scubagoggles\opa_windows_amd64.exe
+Setup: Google API credentials file
+Google credentials (JSON) file [C:\Users\userID\scubagoggles\credentials.json]?
+(WARNING): Google credentials file not found in C:\Users\userID\scubagoggles\credentials.json
+```
+
+### Linux Example
+
+```
+$ scubagoggles setup
+Setup: output directory
+Scubagoggles output directory [/home/userID/scubagoggles]?
+Create directory /home/userID/scubagoggles [Yes/no]?
+  creating: /home/userID/scubagoggles
+  /home/userID/scubagoggles
+Setup: OPA executable directory
+(WARNING): OPA executable not found in PATH
+Location of OPA executable [/home/userID/scubagoggles]?
+  downloading: opa_linux_amd64_static
+  OPA executable: /home/userID/scubagoggles/opa_linux_amd64_static
+Setup: Google API credentials file
+Google credentials (JSON) file [/home/userID/scubagoggles/credentials.json]?
+(WARNING): Google credentials file not found in /home/userID/scubagoggles/credentials.json
+```
+
+## Navigation
+- Continue to [Download the OPA executable](OPA.md)
+- Return to [Documentation Home](/README.md)
```

Comment:
The download and install file has large amount of text which is helpful to understand the process, however from a user's perspective this is confusing (to find instructions within the text) and a lot of information.
The instructions can be simplified with a shorter version with clear steps on what to do next.
The informative text can be saved in separate place for reference.

Reply:
noted in https://github.com/cisagov/ScubaGoggles/issues/531